build.sbt

```diff
@@ -5,10 +5,13 @@ name := "Agora"
 
 organization := "org.broadinstitute"
 
-scalaVersion := "2.11.12"
-dependencyOverrides += "org.scala-lang" % "scala-compiler" % "2.11.12"
+scalaVersion := "2.12.5"
```

Comment:
up for debate: should we skip 2.12.5 and stick with 2.12.4? https://contributors.scala-lang.org/t/scala-2-12-6-coming-soon/1729

Reply:
Fwiw we rolled back from 2.12.5 to 2.12.4

Reply:
Thanks for bringing this up! I don't feel strongly either way. Will roll back to `2.12.4` unless I hear an argument for not to.